packages/subgraph/schema.graphql

```diff
@@ -183,44 +183,44 @@ enum VoteOption {
   No
 }
 
-# ERC20Voting
+# TokenVoting
 
-type ERC20VotingPlugin implements IPlugin @entity {
+type TokenVotingPlugin implements IPlugin @entity {
   "TODO: attributes should be appended to Plugins once plugin is seperated from core"
   id: ID!
   daos: [DaoPlugin!] @derivedFrom(field: "plugin")
-  proposals: [ERC20VotingProposal!]! @derivedFrom(field: "plugin")
+  proposals: [TokenVotingProposal!]! @derivedFrom(field: "plugin")
   relativeSupportThresholdPct: BigInt
   totalSupportThresholdPct: BigInt
   minDuration: BigInt
   proposalCount: BigInt
   token: ERC20Token
-  members: [ERC20VotingVoter!]! @derivedFrom(field: "plugin")
+  members: [TokenVotingVoter!]! @derivedFrom(field: "plugin")
 }
 
-type ERC20VotingVoter @entity {
+type TokenVotingVoter @entity {
   id: ID! # address
   address: String # address as string to facilitate filtering by address on the UI
   proposals: [ERC20Vote!]! @derivedFrom(field: "voter")
-  plugin: ERC20VotingPlugin!
+  plugin: TokenVotingPlugin!
   lastUpdated: BigInt
 }
 
 type ERC20Vote @entity {
```

Comment:
are we going to have a different type for NFTs?

Reply:
No, TokenVoting will be agnostic wheter it is an ERC20 or ERC721 as long as the token supports OZ's `Votes` interface. Our `GovernanceERC20` and `GovernanceWrappedERC20` supports `ERC20Votes` and they also provide an `ERC721Votes`.

See
https://docs.openzeppelin.com/contracts/4.x/api/governance#Votes
https://docs.openzeppelin.com/contracts/4.x/api/token/erc20#ERC20Votes
https://docs.openzeppelin.com/contracts/4.x/api/token/erc721#ERC721Votes

We will have a follow-up task to add `GovernanceERC721` and `GovernanceWrappedERC721`.
https://discord.com/channels/672466989217873929/1045348071942410280/1045705111961358356

Reply:
in this case I suggest we rename `ERC20Vote` type in subgraph too